notebook_examples/example.ipynb

```diff
@@ -35,14 +37,19 @@
     "\n",
     "source = data.stocks()\n",
     "\n",
-    "alt.Chart(source).mark_area(\n",
-    "    color=\"lightblue\",\n",
-    "    interpolate='step-after',\n",
-    "    line=True\n",
-    ").encode(\n",
-    "    x='date',\n",
-    "    y='price'\n",
-    ").transform_filter(alt.datum.symbol == 'GOOG')\n"
+    "\n",
+    "with warnings.catch_warnings():\n",
+    "    warnings.filterwarnings(\"ignore\", message=\"iteritems is deprecated and will be removed in a future version. Use .items instead.\")\n",
```

Comment:
had to catch this because it shows up in the report. I could add the warning to the ground truth but the warning has a timestamp in it that always shows up in the diff.